package com.company.chapter3;

import org.junit.Test;

import static org.junit.Assert.*;

public class SampleTest {

    Sample ut = new Sample();

    @Test
    public void testProc01() throws Exception {
        ut.proc01();
    }

    @Test
    public void testProc02() throws Exception {
        ut.proc02();
    }

    @Test
    public void testProc03() throws Exception {
        ut.proc03();
    }

    @Test
    public void testProc04() throws Exception {
        ut.proc04();
    }

    @Test
    public void testProc05() throws Exception {
        ut.proc05();
    }

    @Test
    public void testProc06() throws Exception {
        ut.proc06();
    }

    @Test
    public void testProc30() throws Exception {
        ut.proc30();
    }

    @Test
    public void testProc31() throws Exception {
        ut.proc31();
    }

    @Test
    public void testProc40() throws Exception {
        ut.proc40();
    }

    @Test
    public void testProc41() throws Exception {
        ut.proc41();
    }

    @Test
    public void testProc42() throws Exception {
        ut.proc42();
    }

    @Test
    public void testProc43() throws Exception {
        ut.proc43();
    }


    @Test
    public void testProc44() throws Exception {
        ut.proc44();
    }

    @Test
    public void testProc441() throws Exception {
        ut.proc441();
    }

    @Test
    public void testProc450() throws Exception {
        ut.proc450();
    }

    @Test
    public void testProc451() throws Exception {
        ut.proc451();
    }
}